traffic_ops/traffic_ops_golang/cachegroup/cachegroups.go

```diff
@@ -163,6 +163,20 @@ func (cg TOCacheGroup) Validate() error {
 		return err
 	}
 
+	if cg.Fallbacks != nil {
+		for _, fallback := range *cg.Fallbacks {
+			isValid, err := cg.isValidCacheGroupFallback(fallback)
+			if err != nil {
+				return err
+			}
+
+			if !isValid {
+				log.Errorf("the cache group " + fallback + "is not valid as a fallback.  It must exist as a cache group and be of type EDGE_LOC.")
```

Comment:
I think this log line can be removed since we don't really need to log details about how a user's input was invalid. That information is really only meant for the user.

Reply:
done